scripts/src/java/org/oppia/android/scripts/maven/RetrieveLicenseTexts.kt

```diff
@@ -20,6 +20,7 @@ import javax.xml.transform.stream.StreamResult
 
 private const val MAVEN_DEPENDENCY_LIST_NOT_UP_TO_DATE =
   "maven_dependencies.textproto is not up-to-date"
+private const val MAX_CHARS_LIMIT = 16383
```

Comment:
Can you mention the reference from where you got this limit?

Reply:
[This](https://stackoverflow.com/a/51733275/12215015) answer says that the max limit is 32,767 bytes and a char takes 2 bytes. So I halved the mentioned size and set it as the limit.